### PR TITLE
edits for apple, MS

### PR DESCRIPTION
**Pull request by TzviyaSiegman**
proposed tweaks based on FOs from MS and recommended changes from Apple, et al

### CODE REVIEW

**Comment by lrosenthol** [comment]
This seems like a LOT of changes.  I thought we had agreed to keep changes
to a minimum to avoid a "re-vote"...

**Comment by TzviyaSiegman** [comment]
@lrosenthol. most of the changes are moving things around. These are responses to formal objections. We are not proposing anything to the IG yet.

**Comment by WSchindler** [comment]
It's a good idea to expand the abbreviations such as WCAG! 

I'm not sure whether replacing "publications" in the mission statement by "Web Publications" precisely covers the intended meaning. Is a "Web Publication" any publication using standard Open Web Platform technologies, even if it is distributed as a zip file for offline use, or is it mainly a publication on the Web? Your definition seems to suggest the first interpretation, but would I go too far to think of any publication - even a print product - as a data resource that should/could use standard Open Web Platform technologies (e.g. Print CSS)? I think "publications" at this section keeps this more open for further developments.

In the definition of the "Web Publication" I'm afraid we have got the same information twice:
> <strong>Definition: Web Publication (WP):</strong> A collection of one or more constituent resources, organized together in a uniquely identifiable grouping that may be presented using standard Open Web Platform technologies is a **collection of one or more constituent resources, organized together in a uniquely identifiable grouping that may be presented using standard Open Web Platform technologies.**

File: index.html [hunk 1]
```diff
@@ -175,12 +176,13 @@ <h1 id="title">Publishing Working Group Charter</h1>
...
+                    <strong>Definition: Web Publication (WP):</strong> A collection of one or more constituent resources, organized together in a uniquely identifiable grouping that may be presented using standard Open Web Platform technologies is a collection of one or more constituent resources, organized together in a uniquely identifiable grouping that may be presented using standard Open Web Platform technologies.
+                    <!--A Web Publication is not just a collection of links.--> The act of publishing involves obtaining resources and organizing them into a publication, which must be “manifested” by having files on a Web server.
+                    Thus the publisher provides an <a href="https://tools.ietf.org/html/rfc6454">origin</a> for the WP, and a URL that can uniquely identify that <a href="https://archive.ifla.org/VII/s13/frbr/frbr1.htm#3.2">(FRBR) (Functional Requirements for Bibliographic Records) manifestation</a>.
```
Comment (mattgarrish) [inline index.html:183]: Minor nit, but couldn't this just be: &lt;abbr title="Functional Requirements for Bibliographic Records"&gt;FRBR&lt;/abbr&gt;

A paren following a paren is strange. Alternatively, maybe wrap the title inside so they're more explicitly linked (FRBR (...)), or drop the parentheses around FRBR.

**Comment by WSchindler** [comment]
I would prefer to keep the paragraph on EPUB and EPUB 4. Why? There is an established EPUB ecosystem which we don't want to lose. It will be the task of the Publishing Working Group to define WP and PWP. In the end it may happen that EPUB 4 is just another name for PWP, but it could be that there will be specific publishing requirements that could only be covered by a PWP with specific features which we could dub "EPUB 4". Being able to cover the full functionality of EPUB 3.1. is in my opinion an important requirement for PWP.

**Comment by TzviyaSiegman** [comment]
Based on discussion with chairs and team, this PR is for reference only. Refer to http://w3c.github.io/dpubwg-charter/ for current draft of charter.

**Pull request closed without merging**